server/src/main/java/org/elasticsearch/action/search/ExpandSearchPhase.java

```diff
@@ -31,13 +30,13 @@
  */
 final class ExpandSearchPhase extends SearchPhase {
     private final SearchPhaseContext context;
-    private final InternalSearchResponse searchResponse;
+    private final SearchHits searchHits;
     private final Supplier<SearchPhase> nextPhase;
 
-    ExpandSearchPhase(SearchPhaseContext context, InternalSearchResponse searchResponse, Supplier<SearchPhase> nextPhase) {
+    ExpandSearchPhase(SearchPhaseContext context, SearchHits searchHits, Supplier<SearchPhase> nextPhase) {
```

Comment:
We only use the hits, so lets only pass around the hits => only have to worry about ref-counting the hits :)